loopin.go

```diff
@@ -756,7 +756,7 @@ func (s *loopInSwap) publishTimeoutTx(ctx context.Context,
 	}
 
 	witnessFunc := func(sig []byte) (wire.TxWitness, error) {
-		return s.htlc.GenTimeoutWitness(sig)
+		return s.htlc.GenTimeoutWitness(s.SenderKey, sig), nil
```

Comment:
For an encapsulation pov, I wouldn't expect the sender key to be required here. The `htlc` object has already been given that key when it was constructed?

Reply:
done